Dockerfile

```diff
@@ -76,7 +83,7 @@ CMD ["--vendor", "--enable-all", "--line-length=120", "./..."]
 #
 # DOCKER_BUILDKIT=1 docker build --target goreleaser --tag ockam/tool/goreleaser:latest .
 # docker run --rm --volume "$(pwd):/project" ockam/tool/goreleaser:latest
-FROM go as goreleaser
+FROM go-alpine as goreleaser
```

Comment:
so goreleaser builds the final binaries this likely needs to be the `go` debian image to fix the release binaries.